__tests__/app.test.js

```diff
@@ -48,3 +48,33 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns a single article with the required properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.article;
+        expect(articles).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
Do you expect any number on this response? If you are fetching an article primarily based on its article id you should assert that what you are receiving back is the article_id you expect.